driftbase/tests/auth/test_gamecenter.py

```diff
@@ -80,9 +80,7 @@ class GameCenterCase(unittest.TestCase):
     def setUp(self):
 
         def requests_get_mock(url, *args, **kw):
-            if url == 'broken url':
-                raise requests.exceptions.RequestException('Url broken - unittest.')
-            elif url == 'broken cert':
+            if 'broken_cert' in url:
                 broken_cert = collections.namedtuple('Response', 'content status_code')
                 broken_cert.content = 'not a valid cert'
                 broken_cert.status_code = 200
```

Comment:
Don't think it matters, but this is weird; namedtuple returns a class object, not an instance of a class, so we're not setting the tuple fields and returning a Response instance, we're setting class attributes and returning the class itself here...

Reply:
Apparently this still works, even if it's technically wrong. The attributes that get set on the class are readable just as if it was an instance. I'll fix it though.